api/cluster/resource/templater.go

```diff
@@ -385,13 +390,32 @@ func (t *InferenceServiceTemplater) enrichStandardTransformerEnvVars(envVars mod
 	addEnvVar := models.EnvVars{}
 	addEnvVar = append(addEnvVar, models.EnvVar{Name: transformerpkg.DefaultFeastSource, Value: t.standardTransformerConfig.DefaultFeastSource.String()})
 
-	// adding feast storage config env variable
+	// adding feast related config env variable
 	feastStorageConfig := t.standardTransformerConfig.ToFeastStorageConfigs()
-
 	if feastStorageConfigJsonByte, err := json.Marshal(feastStorageConfig); err == nil {
 		addEnvVar = append(addEnvVar, models.EnvVar{Name: transformerpkg.FeastStorageConfigs, Value: string(feastStorageConfigJsonByte)})
 	}
 
+	// Add keepalive configuration for predictor
+	// only pyfunc config that enforced by merlin
+	keepAliveModelCfg := t.standardTransformerConfig.ModelClientKeepAlive
+	if modelService.Protocol == protocol.UpiV1 && modelService.Type == models.ModelTypePyFunc {
+		addEnvVar = append(addEnvVar, models.EnvVar{
```

Comment:
Can you make the chane in line 401 to 417 consistent with line 415-417. In particular on how to append en var

Reply:
Is it also possible to override these value by users?

Reply:
yes, user can still modify this